### PR TITLE
Add simple any of renderer

### PR DESCRIPTION
**Pull request by eneufeld**
* this renderer handles only cases where the schema has
an anyof with strings and enums.

### CODE REVIEW

**Comment by eneufeld** [comment]
@edgarmueller I would be interested in suggestions to make it more generic.

File: packages/material/src/controls/MaterialSimpleAnyOfControl.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    id,
+    enabled,
+    uischema,
+    isValid,
+    path,
+    handleChange,
+    schema
+  } = props;
+
+  const findEnumSchema = (schemas: JsonSchema[]) =>
```
Comment (edgarmueller) [inline packages/material/src/controls/MaterialSimpleAnyOfControl.tsx]: Hoist `findEnumSchema` and `findTextSchema` out of render component, re-use in tester.

File: packages/material/test/renderers/MaterialSimpleAnyOfRenderer.test.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    };
+    const store: Store<JsonFormsState> = createStore(
+      combineReducers({ jsonforms: jsonformsReducer() }),
+      s
+    );
+    store.dispatch(Actions.init(testData, testSchema, testUiSchema));
+    return store;
+  };
+
+describe('Material input control', () => {
```
Comment (edgarmueller) [inline packages/material/test/renderers/MaterialSimpleAnyOfRenderer.test.tsx]: Update test name

File: packages/material/src/controls/MaterialSimpleAnyOfControl.tsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+      autoFocus={uischema.options && uischema.options.focus}
+      fullWidth={!mergedConfig.trim || maxLength === undefined}
+      inputProps={inputProps}
+      error={!isValid}
+      endAdornment={dataList}
+    />
+  );
+};
+
+export class MaterialSimpleAnyOfControl extends Control<ControlProps, ControlState> {
```
Comment (edgarmueller) [inline packages/material/src/controls/MaterialSimpleAnyOfControl.tsx]: SimpleAnyOf is a somewhat misleading name, as this renderer is specific to type string with enums.

File: packages/material/src/controls/MaterialSimpleAnyOfControl.tsx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  render() {
+    return (
+      <MaterialInputControl {...this.props} input={MuiAutocompleteInputText} />
+    );
+  }
+}
+const hasEnumAndText = (schemas: JsonSchema[]) => {
```
Comment (edgarmueller) [inline packages/material/src/controls/MaterialSimpleAnyOfControl.tsx]: Replace 'text' wording with 'string'

**Pull request merged**